client/src/pages/Home.js

```diff
@@ -1,13 +1,17 @@
-import React, { useContext, useEffect } from "react";
-import styled from "styled-components";
-import { Link } from "react-router-dom";
-import { PageContainer } from "../styles/index.js";
-import Carousel from "../components/Carousel.js";
+import React, { useContext, useEffect } from 'react';
+import styled from 'styled-components';
+import { Link } from "react-router-dom"
+
+import { PageContainer } from '../styles/index.js';
+import Carousel from '../components/Carousel.js';
+import HomeItemsWidget from '../components/item/HomeItemsWidget';
 
 function Home({ state, dispatch, init }) {
   const { dev, user, HomeData } = state;
   const { lat, lng, local } = user;
 
+
```

Comment:
I think this is an older version of the home page file than what dev currently is. I would just ask that you look at the current dev branch and make the widget changes you need on that. Thanks @MarkAlperin

Reply:
This can be fixed via the merge conflict for this file. See below @MarkAlperin